core/src/test/scala/org/apache/spark/sql/delta/DeleteSQLSuite.scala

```diff
@@ -56,7 +56,7 @@ class DeleteSQLSuite extends DeleteSuiteBase  with DeltaSQLCommandTest {
       withTempView("v") {
         Seq((1, 1), (0, 3), (1, 5)).toDF("key", "value").write.format("delta").saveAsTable("tab")
         sql("CREATE TEMP VIEW v AS SELECT * FROM tab")
-        sql("DELETE FROM v WHERE key = 1 AND VALUE = 5")
+        checkAnswer(sql("DELETE FROM v WHERE key = 1 AND VALUE = 5"), Row(1))
```

Comment:
nit: `VALUE` lowercase for consistency

Reply:
Will do. I am surprised I didn't find tests for non views, am I missing something?